env.d/development/dev

```diff
@@ -5,3 +5,6 @@ AUTHENTICATION_BACKEND=base
 # LMS Backend
 EDX_BASE_URL=http://localhost:8073
 EDX_BACKEND=richie.apps.courses.lms.base.BaseLMSBackend
+
+# LTI Backend
+LTI_TEST_BASE_URL=http://localhost:8060/lti/videos/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}
```

Comment:
do you need to repeat it here as you already set it in the common env file?

Reply:
I didn't look how and when env files are loaded, so I put it everywhere :sweat_smile: 
I'll check it.

Reply:
I'm asking the same question about circleci environment variables?